pkg/minikube/registry/drvs/vfkit/vfkit.go

```diff
@@ -41,13 +42,17 @@ const (
 )
 
 func init() {
+	priority := registry.Unknown
+	if runtime.GOOS == "darwin" {
+		priority = registry.Preferred
```

Comment:
I think is better than preffered registry.Default so it be the same level as qemu,